get-thin-edge_io.sh

```diff
@@ -134,11 +134,11 @@ esac
 rm -R /tmp/tedge
 
 # Test if tedge command is there and working
-tedge help >/dev/null
-if [ $? -eq 0 ]; then
-    echo "\n${BLUE}thin-edge.io is now installed on your system!${COLORRESET}"
+if tedge help >/dev/null;
+then
+    echo -e "\n${BLUE}thin-edge.io is now installed on your system!${COLORRESET}"
     echo ""
-    echo "You can go to our documentation to find next steps:${BLUE} https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/003_registration.md ${COLORRESET}"
+    echo -e "You can go to our documentation to find next steps:${BLUE} https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/003_registration.md ${COLORRESET}"
 else
-    echo "Something went wrong in the installation process please try the manual installation steps instead:\n https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/002_installation.md"
+    echo -e "Something went wrong in the installation process please try the manual installation steps instead:\n https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/002_installation.md"
```

Comment:
In order to be `sh` compatible.

```suggestion
    echo "Something went wrong in the installation process please try the manual installation steps instead:"
    echo "     https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/howto-guides/002_installation.md"
```